docs/articles/clustering/node-roles.md

```diff
@@ -0,0 +1,24 @@
+---
+uid: node-roles
+title: Node Roles
+---
+
+# Node Roles
+
+Not all nodes of a cluster need to perform the same function. For example, there might be one sub-set which runs the web front-end, one which runs the data access layer and one for the number-crunching. Choosing which actors to start on each node, for example cluster-aware routers, can take node roles into account to achieve this distribution of responsibilities.
```

Comment:
Need to expand this:

* Why are roles important?
* How are they used elsewhere in different parts of Akka.Cluster (this should be a bulleted list) ?
* What do they allow end-users to do?